data/data-pipeline/data_pipeline/score/score_d.py

```diff
@@ -0,0 +1,29 @@
+from data_pipeline.score.score import *
+
+
+class ScoreD(Score):
+    def add_columns(self) -> pd.DataFrame:
+        logger.info("Adding Scores D and E")
```

Comment:
Ik ept scores d and e together because they use the same fields and mostly the same logic, so it didn't seem worth splitting out and moving shared logic to the parent class (which would decrease code legibility).

Reply:
Makes sense!!